### PR TITLE
New dev indexer-dhstore-dhfind set

### PR DESCRIPTION
**Pull request by gammazero**
Create a new set of indexers in dev. Each indexer has a dedicated dhstore and a dedicated dhfind. Their names are matched, for example: `alva-indexer`, `dhstore-alva`, `dhfind-alva`

The assigner configuration has changed to use the new dev indexers as its indexer pool.

### CODE REVIEW

File: deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/cora/pvc_data.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    app: indexer-single
+    app.kubernetes.io/managed-by: kustomization
+    app.kubernetes.io/part-of: storetheindex
+  name: data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti
```
Comment (masih) [inline deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/cora/pvc_data.yaml:14]: This volume is used for provider info only right? 

Would it make sense to look at its utilisation in other nodes and potentially reduce this? We can always grow it later but shrinking it is non trivial and would need data migration.

**Review by masih — APPROVED** [review]
@gammazero the infra ninja 🥷

**Review by ischasny — APPROVED** [review]

File: deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/cora/pvc_data.yaml (hunk 1 above)
Comment (gammazero) [reply deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/cora/pvc_data.yaml:14]: The data stores do grow with recording all the ads ingested as well as keeping the mh counts for all contextIDs. ... and of course all that wonderful data-transfer FSM state.  On ago this is currently 24G... so would 100G be good?

Comment (masih) [reply deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/cora/pvc_data.yaml:14]: `100Gi` sounds good 👍

**Comment by gammazero** [comment]
Closing in favor of new indexers deployed for use with FDB.  If we want a set indexer-dhstore-dhfind nodes later, that can be done in a new PR.

**Pull request closed without merging**